setup.py

```diff
@@ -31,5 +31,8 @@
     entry_points={
         'console_scripts': ['apertium-streamparser=streamparser:main'],
     },
+    package_data={
+        'streamparser': ['py.typed'],
+    },
     py_modules=['streamparser'],
```

Comment:
You need to add `zip_safe=False` otherwise setuptools will make an egg and mypy doesn't work with that.

Reply:
Thanks for the suggestion! I tried this out and didn't have much luck.

```
$ make -B dist
python3 setup.py sdist
running sdist
running egg_info
...
creating apertium-streamparser-5.0.3
creating apertium-streamparser-5.0.3/apertium_streamparser.egg-info
copying files to apertium-streamparser-5.0.3...
copying LICENSE -> apertium-streamparser-5.0.3
copying MANIFEST.in -> apertium-streamparser-5.0.3
copying README.md -> apertium-streamparser-5.0.3
copying py.typed -> apertium-streamparser-5.0.3
copying setup.py -> apertium-streamparser-5.0.3
copying streamparser.py -> apertium-streamparser-5.0.3
...
Creating tar archive
removing 'apertium-streamparser-5.0.3' (and everything under it)
```

```
$ pip3 install /Users/Sushain/Documents/streamparser/dist/apertium-streamparser-5.0.3.tar.gz
...
Successfully built apertium-streamparser
Installing collected packages: apertium-streamparser
Successfully installed apertium-streamparser-5.0.3
```

```
$ python3
>>> import streamparser
>>> streamparser.parse
<function parse at 0x104b36950>
```

```
$ cat $(which mypy)
#!/usr/local/opt/python/bin/python3.7

# -*- coding: utf-8 -*-
import re
import sys

from mypy.__main__ import console_entry
...
````

```
$ cat test.py
import streamparser

reveal_type(streamparser.parse)
```

```
$ mypy test.py
test.py:1: error: Cannot find module named 'streamparser'
test.py:1: note: (Perhaps setting MYPYPATH or using the "--ignore-missing-imports" flag would help)
test.py:3: error: Revealed type is 'Any'
```

Reply:
Alas, I had a suspicion single-file modules and `py.typed` wouldn't cooperate. The best strategy is probably to just create a real (temporary) package with the source at make-time (just to keep this repo from having a somewhat overkill `src` folder).

Thanks for your help and confirming this strategy doesn't work!